data/quarg/quarg outfits.txt

```diff
@@ -140,4 +146,5 @@ outfit "Quantum Shield Generator"
 	"outfit space" -120
 	"shield generation" 10
 	"shield energy" 10
+	"shield protection" 0.1
```

Comment:
Sorry github didnt tell me about your review for some reason
Anyway, sounds good to me, what about 8 gen and energy?

Reply:
I would, personally, drop it down to 6 shield generation. 
That gives it the best regen/mass in the game against anything as strong as, or stronger than a stock punisher while being worse than even Hai/Human versus t1 ships. 
Very much specialized in fighting other high-tier ships.

Reply:
I don't see a need to drop it. Being *awesome* against high turret weapons while merely being great against lots tier sounds perfectly acceptable to me.

Reply:
Piercing protection I'd say no because the arfecta disables these by taking the hull out
But disruption now that makes sense for sure

Reply:
I added piercing and discharge (not used yet) resistance that consume energy to work (no discharge dmg in the game currently but still) and removed the protection since I don't really like passive things honestly
I also made the shields better and more efficient, as they should be I think. (bit more gen and bit less cost so they're above human in efficiency)

Reply:
I cutted it down by 5

Reply:
I don't see piercing resistance in the data, but I do think it fits:
A) the most serious and regular combat we see Quarg engage in is vs automata, so it makes sense they would work on countering their most common foe.
B) You mention the Arfecta using piercing to disable Quarg ships. Given that there Quarg have already lost one major conflict with the Pug, which is more likely? That the Quarg have learned from it and developed new measures to give themselves a chance in a future conflict, or that the Pig have put in place new offense so they can win more easily next time? The latter seems out of character for the pug, so I think it makes the most sense to assume that their piercing contributed to their victory, so it makes sense that the Quarg would actively take steps to improve their odds in the future. After all, the Pug are the single most serious that in the game to the Quarg, so their defensive research is likely focused on defeating their previously decisive advantages.

Reply:
Makes sense. I didnt add more than 5% to not make it op or anything since it is directly reduced from the weapon shooting it I think

Reply:
That seems reasonable. We can always adjust it later if necessary, but this seems like a good amount.